pkg/build/api/validation/validation_test.go

```diff
@@ -144,7 +143,10 @@ func TestValidateBuildUpdate(t *testing.T) {
 	}
 }
 
-func TestBuildConfigValidationSuccess(t *testing.T) {
```

Comment:
no we can probably live without that.. if we manage to break successful validation i think we'll find out pretty fast :)